zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/topology/TopologyClusterTest.java

```diff
@@ -76,23 +122,27 @@ private void assertPartitionInTopology(final List<BrokerInfo> brokers, final int
             PartitionBrokerRole.LEADER, PartitionBrokerRole.FOLLOWER, PartitionBrokerRole.FOLLOWER);
   }
 
-  @Test
-  public void shouldExposeClusterSettings() {
-    // when
-    final Topology topology = CLIENT_RULE.getClient().newTopologyRequest().send().join();
-
-    // then
-    assertThat(topology.getClusterSize()).isEqualTo(CLUSTERING_RULE.getClusterSize());
-    assertThat(topology.getPartitionsCount()).isEqualTo(CLUSTERING_RULE.getPartitionCount());
-    assertThat(topology.getReplicationFactor()).isEqualTo(CLUSTERING_RULE.getReplicationFactor());
-    // NOTE: this fails in Intellij because we don't have access to the package version but it works
-    // when run from the CLI
-    assertThat(topology.getGatewayVersion())
-        .isEqualTo(Gateway.class.getPackage().getImplementationVersion());
-
-    for (final BrokerInfo broker : topology.getBrokers()) {
-      assertThat(broker.getVersion())
-          .isEqualTo(Broker.class.getPackage().getImplementationVersion());
+  private static ZeebeClient createZeebeClient() {
+    final var gateway = CLUSTER.anyGateway();
+    return CLUSTER
+        .newClientBuilder()
+        .gatewayAddress(gateway.gatewayAddress())
+        .gatewayRestApiPort(gateway.mappedPort(TestZeebePort.REST))
```

Comment:
👍 This is cool, this returns the current `server.port` configured, right? Which basically is the REST port.

Reply:
@tmetzke yes, this was already implemented by @npepinpe with the REST API server infrastructure.